### PR TITLE
Hybridauth v3.0

### PR DESCRIPTION
**Pull request by pierrestudios**

### CODE REVIEW

File: hybridauth/library/src/Provider/Facebook.php [hunk 1]
```diff
@@ -47,7 +47,7 @@ class Facebook extends OAuth2
     /**
      * {@inheritdoc}
      */
-    protected $apiBaseUrl = 'https://graph.facebook.com/v2.8/';
+    protected $apiBaseUrl = 'https://graph.facebook.com/v2.12/';
```
Comment (sviluppomania) [inline hybridauth/library/src/Provider/Facebook.php]: You must change the last active version. The V3.3.

**Comment by sviluppomania** [comment]
What is this update for?

**Comment by mattrabe** [comment]
@sviluppomania master branch currently uses a 3.0-Release-Candidate version of hybridauth library. This PR brings it up to 3.0

File: hybridauth/library/src/Provider/Facebook.php (hunk 1 above)
Comment (pierrestudios) [reply hybridauth/library/src/Provider/Facebook.php]: Code is updated to use Facebook API Version 3.3

**Pull request closed without merging**